modules/pages/client/config/pages.client.routes.js

```diff
@@ -169,7 +169,18 @@ function PagesRoutes($stateProvider) {
       footerHidden: true,
     });
   } else {
-    $stateProvider.state('home', { url: '/' });
+    $stateProvider.state('home', {
+      url: '/',
+      controller: 'HomeController',
+      controllerAs: 'stats',
+      resolve: {
+        // A string value resolves to a service
+        SettingsService: 'Home',
```

Comment:
This should be `UserCountService: 'UserCountService'` to work. (It's some Angular weirdness)